docs/_custom-rule-tutorial-code/eslint.config.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+// Import the ESLint plugin
+const eslintPluginFooBar = require("./eslint-plugin-foo-bar");
+
+module.exports = [
+    {
+        files: ["**/*.js"],
+        languageOptions: {
+            sourceType: "commonjs",
+            ecmaVersion: "latest",
+        },
+        // Using the eslint-plugin-foo-bar plugin downloaded from npm
+        plugins: {"foo-bar": eslintPluginFooBar},
+        rules: {
+            "foo-bar/foo-bar": "error",
```

Comment:
I think this may be a bit too cheeky for the example. Maybe we can use a plugin name like `eslint-plugin-example` and a rule name of `enforce-foo-bar` so this becomes `example/enforce-foo-bar`.